src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/action/CreateIssueAction.kt

```diff
@@ -24,6 +25,7 @@ class CreateIssueAction(
             val dashboardPage = meter.measure(VIEW_DASHBOARD) {
                 jira.goToDashboard().waitForDashboard()
             }
+            dismissAllJiraPopups(jira.driver)
```

Comment:
I am not sure if those popups shall be closed before opening the creation dialog. I will give it a try to see if the CI passes.

Reply:
the CI passed if dismissing popups before open the creation dialog, I will test it in HWR today.